sql3/test/defs/defs_set_functions.go

```diff
@@ -304,3 +304,93 @@ var setParameterTests = TableTest{
 		},
 	},
 }
+
+// test IDSetQ and StringSetQ functions
+var setTimeQuantumTests = TableTest{
+	name: "selectwithsetqliterals",
+	Table: tbl(
+		"selectwithsetqliterals",
+		srcHdrs(
+			srcHdr("_id", fldTypeID),
+			srcHdr("a", fldTypeInt, "min 0", "max 1000"),
+			srcHdr("b", fldTypeInt, "min 0", "max 1000"),
+			srcHdr("ssq1", fldTypeStringSetQ, "timequantum 'YMD'"),
+			srcHdr("isq1", fldTypeIDSetQ, "timequantum 'YMD'"),
+		),
+	),
+	SQLTests: []SQLTest{
+		// can't find example syntax or docs for how to put time quantum fields in with srcRow()
+		// so i'm inserting them with SQL statements.
```

Comment:
Appreciate this! In some situations I think using SQL to setup test data for a SQL test is a better approach. Once I spent considerable time scratching my head due to a faulty insert generated by the test framework.

Reply:
I'd prefer we update the test framework to handle time quantum fields, rather than start doing inserts in the result tests.

Reply:
Probably a good idea at some point, but at a quick look, I think it'd be enough work to require its own ticket.